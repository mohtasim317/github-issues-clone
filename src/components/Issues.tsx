import React, { FC } from "react";
import "../index.css";

interface Props {
  key: number;
  title: string;
  // number: number;
  // id: number;
}

const Issues: FC<Props> = (props) => {
  return (
    <div className="issue">
      <p>{props.title}</p>
      {/* <h1>{props.id}</h1> */}
    </div>
  );
};

export default Issues;
