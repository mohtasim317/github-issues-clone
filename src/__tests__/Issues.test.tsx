import React from "react";
import renderer from "react-test-renderer";
import Issues from "../components/Issues";

describe("Component", () => {
  it("is pretty cool", () => {
    const component = renderer
      .create(
        <Issues key={1} login={"H"} title={"e"} number={3} created_at={"s"} />
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
