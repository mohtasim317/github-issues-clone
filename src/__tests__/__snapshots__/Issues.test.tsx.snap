// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component is pretty cool 1`] = `
<div
  className="issue"
>
  <div
    className="svg-and-issue"
  >
    <svg
      aria-hidden="true"
      className="svg open-issue-svg"
      version="1.1"
      viewBox="0 0 16 16"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M8 1.5a6.5 6.5 0 100 13 6.5 6.5 0 000-13zM0 8a8 8 0 1116 0A8 8 0 010 8zm9 3a1 1 0 11-2 0 1 1 0 012 0zm-.25-6.25a.75.75 0 00-1.5 0v3.5a.75.75 0 001.5 0v-3.5z"
        fill-rule="evenodd"
      />
    </svg>
    <h4
      className="issue-title"
    >
      e
    </h4>
  </div>
  <p
    className="issue-creator"
  >
    #
    3
     opened 
    Invalid date
     by 
    H
  </p>
</div>
`;
